src/lib.rs

```diff
@@ -0,0 +1,420 @@
+extern crate colored;
+extern crate git2;
+extern crate strum;
+extern crate strum_macros;
+
+use colored::*;
+use git2::Repository;
+use std::{
+    collections::HashMap,
+    ffi::OsStr,
+    fs,
+    process::{Command, Stdio},
+    result,
+};
+
+mod commit_info;
+mod error;
+mod language;
+use commit_info::CommitInfo;
+use error::{Error, Result};
+use language::Language;
+
+fn get_languages_stat(languages: &tokei::Languages) -> Option<HashMap<Language, f64>> {
+    let mut stats = HashMap::new();
+
+    let sum_language_code: usize = languages.iter().map(|(_, v)| v.code).sum();
+
+    if sum_language_code == 0 {
+        None
+    } else {
+        for (k, v) in languages.iter() {
+            let code = v.code as f64;
+            stats.insert(
+                Language::from(*k),
+                (code / sum_language_code as f64) * 100.00,
+            );
+        }
+        Some(stats)
+    }
+}
+
+fn project_license(dir: &str) -> Result<String> {
+    let output = fs::read_dir(dir)
+        .map_err(|_| Error::ReadDirectory)?
+        .filter_map(result::Result::ok)
+        .map(|entry| entry.path())
+        .filter(
+            |entry| {
+                entry.is_file()
+                    && entry
+                        .file_name()
+                        .map(OsStr::to_string_lossy)
+                        .unwrap_or_else(|| "".into())
+                        .starts_with("LICENSE")
+            }, // TODO: multiple prefixes, like COPYING?
+        )
+        .map(|entry| {
+            license::Kind::from_str(&fs::read_to_string(entry).unwrap_or_else(|_| "".into()))
```

Comment:
```
no variant or associated item named `from_str` found for type `license::kind::Kind` in the current scope: variant or associated item not found in `license::kind::Kind`
```

Reply:
`Kind::from_str` is implemented by the `std::str::FromStr` trait. If `FromStr` is not in scope, then foreign types will not have functions that the trait requires. So try adding `use::std::FromStr`.

Just curious, what version of Cargo are you using? It's usually smart enough to know which available traits you need to bring into scope to make a function available.